README.md

```diff
@@ -75,14 +75,7 @@ Gather feedback where uncertainty exists. We are our own best testing team and a
 
 “Problem-solving leaders have one thing in common: a faith that there’s always a better way.” – Gerald M. Weinberg
 
-We expect all our engineers to be leaders on this one, when faced with a problem or when identifying a possible problem, all our engineers are incentivized to apply a proactive approach towards helping find solutions over raising problems for someone else to fix. The guideline below lists a set of steps that can be applied to any problem:
-- Inception, identify the stakeholders, the multiple points of view, and mitigate any communication issues, be transparent about your doubts
-- Negotiation, identify the scope of the problem, some solutions may have pitfalls, but it's important to understand the use cases that they address first
-- Elaboration, with the foundation in place, analyze possible solutions or alternative solutions for an already proposed solution
-- Negotiation is the cooperation phase, you should now engage with other team members and reach for feedback about the possible solutions you found to a problem or an incomplete solution
-- Specification, finally after coming up with an agreement on a solution, modify an existing RFC (Request for Comments) or produce a new one
-- Validation, share your RFC and gather enough feedback to produce a PoC (Proof of Concept). We incentive the should don't tell culture.
-- Requirement Management, once you have all the above done, you have all the data you need to organize work into actionable backlog items.
```

Comment:
The change is from **having** a guideline, to **not having** a guideline a let people figure it out. I believe any smart fellow will always appreciate a guideline and be smart enough to understand which point to skip and which points to keep. As the text suggested it is a "guideline" that "can be applied".

Reply:
But if we don't feel in sync with this, and we would rather remove it I am ok with the change. 👍

Reply:
Reading the document in its entirety it feels out of place to include a structured step-by-step process for problem solving in the context of general principles.

@jpinho is there a way to maybe condense some of the ideas from the guidelines into a sentence or two?

Mostly what bothers me here is that it’s easy to interpret this guideline as a process we are asking everyone to follow

Reply:
i totally agree. I think finding solutions or solving proactively issues is better than following a guide. Smart ppl will find out :)

Reply:
Sure, I get what you mean about the "feels like" -> "out of place".
I'm fine with getting rid of it, and add something later when we find out something that is more compact and less prescriptive.